src/azure-cli/setup.py

```diff
@@ -98,7 +98,7 @@
     'azure-mgmt-iothubprovisioningservices==1.1.0',
     'azure-mgmt-keyvault==9.3.0',
     'azure-mgmt-kusto~=0.3.0',
-    'azure-mgmt-loganalytics~=13.0.0b2',
+    'azure-mgmt-loganalytics==13.0.0b2',
```

Comment:
https://github.com/Azure/azure-cli/pull/20968 is the first time to use unpinned beta version. @PARADISSEEKR